lala_bar/src/main.rs

```diff
@@ -87,6 +89,10 @@ pub enum Message {
     Ready(Sender<NotifyCommand>),
     #[allow(unused)]
     LinkClicked(markdown::Url),
+    Close(iced::window::Id),
```

Comment:
Do we need this event? I think it is  better  to  change the event to toggle calendar